src/components/SidebarTopButton.stories.tsx

```diff
@@ -32,18 +33,32 @@ export const Outdated: Story = {
   },
 };
 
+const WithProgress = (props: ComponentProps<typeof SidebarTopButton>) => {
+  const [buildProgressPercentage, setProgress] = useState(20);
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setProgress((p) => (p < 100 ? p + 1 : 0));
+    }, 300);
+    return () => clearInterval(interval);
+  }, []);
+  return (
+    <SidebarTopButton
+      {...props}
+      isRunning
+      localBuildProgress={{
+        ...INITIAL_BUILD_PAYLOAD,
+        buildProgressPercentage,
+        currentStep: "build",
+      }}
+    />
+  );
+};
+
 export const IsRunning: Story = {
-  args: {
-    isRunning: true,
-    localBuildProgress: {
-      ...INITIAL_BUILD_PAYLOAD,
-      buildProgressPercentage: 40,
-      currentStep: "build",
-    },
-  },
+  render: WithProgress,
```

Comment:
Isn't this going to lead to inconsistency in the Chromatic snapshot?

Reply:
I ran this through 3 builds and they all ran consistently. I'll add an isChromatic check regardless.